Test/Load/setup-files/README.md

```diff
@@ -0,0 +1,35 @@
+# Load Test Setup 
+1. Setup M2 store on LightSail: [M2 LightSail Setup](https://www.notion.so/boltteam/Remote-dev-server-with-lightsail-8a053570b68c4ac78561cf04cbde8405)
+
+2. Configure your M2 store with Bolt: [M2 Bolt Configuration](https://docs.bolt.com/docs/magento-2-integration#section-2-plugin-configuration)
+
+3. Add the line below in `bolt-magento2/etc/di.xml`: 
+	```
+	<preference for="Bolt\Boltpay\Api\LoadTestCartDataInterface" type="Bolt\Boltpay\Model\Api\LoadTestCartData" />
+	```
+	Add it below the comment: 
+	```
+	<!-- For rest api hook integration -->
```

Comment:
Was this commented added in a PR? I must've missed it

Reply:
I'm not to sure which PR it was added in but I can see it in master: https://github.com/BoltApp/bolt-magento2/blob/master/etc/di.xml#L37